volatility/framework/plugins/windows/registry/hivescan.py

```diff
@@ -4,16 +4,21 @@
 
 from typing import Iterable
 
-from volatility.framework import renderers, interfaces
+from volatility.framework import renderers, interfaces, symbols
 from volatility.framework.configuration import requirements
 from volatility.framework.renderers import format_hints
-from volatility.plugins.windows import poolscanner
+from volatility.plugins.windows import poolscanner, bigpools
 
 
 class HiveScan(interfaces.plugins.PluginInterface):
     """Scans for registry hives present in a particular windows memory
     image."""
 
+    _version = (1, 0, 0)
+
+    is_windows_8_1_or_later = poolscanner.os_distinguisher(version_check = lambda x: x >= (6, 3),
```

Comment:
We probably want to keep these all in one place and pull them when needed?  It's fine keeping them here for now, but as some point we should pull all the distinguishers out into a central place...